src/definition.cpp

```diff
@@ -395,13 +395,13 @@ void DefinitionImpl::addSectionsToIndex()
 
 void DefinitionImpl::writeDocAnchorsToTagFile(TextStream &tagFile) const
 {
-  if (!m_impl->sectionRefs.empty())
-  {
+#define VECTOR_CONTAINS(x, y) (std::find(x.begin(), x.end(), y) != x.end())
```

Comment:
As this define is used multiple times wouldn't it be better to have it as a global lambda or (template) function or in a global include file?